ratelimit/redis.go

```diff
@@ -120,78 +117,70 @@ func (c *clusterLimitRedis) startSpan(ctx context.Context, spanName string) func
 //
 // If a context is provided, it uses it for creating an OpenTracing span.
 func (c *clusterLimitRedis) AllowContext(ctx context.Context, clearText string) bool {
-	s := getHashedKey(clearText)
 	c.metrics.IncCounter(redisMetricsPrefix + "total")
+	now := time.Now()
+	finishSpan := c.startSpan(ctx, allowSpanName)
+
+	allow, err := c.allow(ctx, clearText)
+	failed := err != nil
+
+	finishSpan(failed)
+	c.measureQuery(allowMetricsFormat, allowMetricsFormatWithGroup, &failed, now)
```

Comment:
with this simplification (dropping the defer), we can also change the *bool argument to plain bool

Reply:
yes, I kept `measureQuery` as is for another round.